worktrolly-ui/src/app/body/dashboard/dashboard.component.ts

```diff
@@ -105,6 +105,11 @@ export class DashboardComponent implements OnInit {
     this.currentSprintName = "Backlog";
   }
 
+  currentSprint(){
+    this.getCurrentSprint();
+    this.currentSprintName = "S" + this.currentSprintNumber;
```

Comment:
I think name is already being set in line 77, it can be removed from here.

Reply:
updated